packages/docusaurus-plugin-content-docs/src/plugin-content-docs.d.ts

```diff
@@ -356,6 +361,11 @@ declare module '@docusaurus/plugin-content-docs' {
     pagination_prev?: string | null;
     /** Should this doc be excluded from production builds? */
     draft?: boolean;
+    /** Allows overriding the last updated author and/or date. Date can be any
+     * parsable date string.
+     * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse
+     */
+    last_update?: FileChange;
```

Comment:
Let's separate this. For the JSDoc of `last_update`, only mention "Allows overriding the last updated author and/or date", and move "Date can be any
parsable date string" to the description of the `FileChange.date` field.

Also, you can use Markdown in JSDoc, so feel free to continue using Markdown links if you think that makes more sense than `@see` tags (which I think they do here)